OverlayPlugin.Core/ClipboardTechSupport.cs

```diff
@@ -20,6 +21,8 @@ namespace RainbowMage.OverlayPlugin
     // so that folks don't have to paste screenshots in discord.
     class ClipboardTechSupport
     {
+        readonly string ACTPathPattern = @"C:\\Users\\.*\\AppData\\Roaming\\Advanced Combat Tracker";
```

Comment:
```suggestion
        readonly string ACTPathPattern = @"[A-Z]:\\Users\\.*\\AppData\\Roaming\\Advanced Combat Tracker";
```

Reply:
Theoretically, Windows could be installed on a different drive.  (I'm assuming ACT uses %APPDIR% during installation rather than having C: hard-coded, although I haven't checked.)

Reply:
Wouldn't it be better to use the actual value of %APPDATA%, i.e. find `%APPDATA%\Advanced Combat Tracker` and replace with `<ACT Folder>`?

Reply:
should all be fixed in new commit